modules/diskq/qdisk.c

```diff
@@ -181,6 +186,55 @@ _create_file(const gchar *filename)
   return TRUE;
 }
 
+static gboolean
+_grab_filename_locks(const gchar *dir, gint *fd)
+{
+  gchar *flock_file_path = g_strdup_printf("%s/%s", dir, FLOCK_FILENAME);
```

Comment:
I think using `g_build_path(G_DIR_SEPARATOR_S, ...` would be a bit better (portable separator and handles a few corner cases when building paths).

Reply:
Thanks, fixed!